assets/src/scripts/monitoring-location/components/hydrograph/parameters.js

```diff
@@ -17,109 +15,218 @@ import {getAvailableParameters} from './selectors/parameter-data';
+        .attr('class', 'grid-row grid-row-inner');
+    const radioButtonDiv = gridRowInnerWithRadioButton.append('div')
+        .attr('class', 'grid-col-1 radio-button__param-select')
+        .append('div')
+        .attr('class', 'usa-radio');
+    radioButtonDiv.append('input')
+        .attr('class', 'usa-radio__input')
+        .attr('id', `radio-${parameter.parameterCode}`)
+        .attr('type', 'radio')
+        .attr('name', 'parameter-selection')
+        .attr('value', `${parameter.parameterCode}`)
+        .call(link(store, (inputElem, selectedParameterCode) => {
+            inputElem.property('checked', parameter.parameterCode === selectedParameterCode ? true : null);
+        }, getSelectedParameterCode));
+    radioButtonDiv.append('label')
+        .attr('class', 'usa-radio__label')
+        .attr('for', `radio-${parameter.parameterCode}`);
+    gridRowInnerWithRadioButton.append('div')
+        .attr('class', 'grid-col-7 description__param-select')
+        .append('div')
+        .text(`${parameter.description}`);
+    gridRowInnerWithRadioButton.append('div')
+        .attr('class', 'grid-col-auto period-of-record__param-select')
+        .text(`${parameter.periodOfRecord.begin_date} to ${parameter.periodOfRecord.end_date}`);
+    const radioRowExpansionControlDiv = gridRowInnerWithRadioButton.append('div')
+        .attr('class', 'grid-col open-close-radio_button_row');
+    drawRowExpansionControl(radioRowExpansionControlDiv, parameter, 'desktop');
+};
+
+/*
+* Helper function that draws a row containing the controls for the WaterAlert subscription.
+* @param {String} siteno - A unique identifier for the monitoring location
+* @param {Object} Element - The target element to append the row
+* @param {Object} parameter - Contains details about the current parameter code
+*/
+const drawWaterAlertRow = function(siteno, element, parameter) {
+    const gridRowInnerWaterAlert = element.append('div')
+        .attr('class', 'grid-row grid-row-inner');
+
+        gridRowInnerWaterAlert.append('div')
+            .attr('id', `wateralert-row-${parameter.parameterCode}`)
+            .attr('class', 'grid-col grid-offset-1 wateralert-row')
+            .append('a')
+            .attr('href', `${config.WATERALERT_SUBSCRIPTION}/?site_no=${siteno}&parm=${parameter.parameterCode}`)
+            .attr('target', '_blank')
+            .attr('class', 'water-alert-cell usa-tooltip')
             .attr('data-position', 'left')
             .attr('data-classes', 'width-full tablet:width-auto')
-            .attr('title', d.waterAlert.tooltipText)
-            .text(d.waterAlert.displayText);
+            .attr('title', parameter.waterAlert.tooltipText)
+            .text(parameter.waterAlert.displayText);
+};
+
+/*
+* A main function that creates the parameter selection list
+* @param {Object} Store - The application Redux state
+* @param {Object} Element - The target element to append the selection list
```

Comment:
Should be first and parameter names should be container (and store)

Reply:
Cool. I changed all the 'container' and 'element' names to 'elem' since that seemed like the most common use in the application.

Reply:
I've actually started using container when it should be a div. I'll use svg or group when it should be an svg/group. If you look in download-data.js or select-actions.js I think I've used container exclusively

Reply:
Will do. Personally, I'm not that fond of 'elem'.